cloud/aws/pgadmin/build-pgdmin

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/bash
+
+# Builds the customized pgadmin Docker image.
+
+docker build \
+  -t docker.io/avritt/pgadmin:avritt-dev \
+  -f pgadmin.Dockerfile .
+
```

Comment:
Replace avritt with civiform, add a github action to build new images with latest tag

Reply:
Done.